client/flutter/lib/api/content/content_store.dart

```diff
@@ -130,11 +130,11 @@ abstract class _ContentStore with Store implements Updateable {
         questionsAnswered2?.bundle?.contentVersion) {
       questionsAnswered = questionsAnswered2;
     }
-    final symptomChecker2 = SymptomCheckerContent(
-        await service.load(locale, countryIsoCode, 'symptom_checker'));
-    if (symptomChecker?.bundle?.contentVersion !=
-        symptomChecker2?.bundle?.contentVersion) {
-      symptomChecker = symptomChecker2;
+    final symptomPoster2 = PosterContent(
+        await service.load(locale, countryIsoCode, 'symptom_poster'));
+    if (symptomPoster?.bundle?.contentVersion !=
+        symptomPoster2?.bundle?.contentVersion) {
+      symptomPoster = symptomPoster2;
```

Comment:
This loads the content serially rather than in parallel?

It bothers me that there's so much duplicated code from Line 91 to 137. I'm not sure if the logic can be abstracted out easily. Not a blocker on commit but please consider refactoring if it's relatively straight forward.

Reply:
Unfortunately its not straightforward.  Dart doesn't allow you to pass setters as functions, so its not really possible.